src/components/guides/issue-panel.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { FunctionComponent, useContext } from "react";
+import { ShepherdOptionsWithType, ShepherdTour, ShepherdTourContext, Tour } from "react-shepherd";
+import { InformationButton } from "../common/button";
+
+const Button = () => {
+  const tour = useContext(ShepherdTourContext);
+  return tour && <InformationButton onClick={tour.start} />;
+};
+
+export const IssueInformationPanel: FunctionComponent = () => {
+  const guideSteps: Array<ShepherdOptionsWithType> = [introduction, stepOne, stepTwo, stepThree, stepComplete];
+  const tourOptions: Tour.TourOptions = {
+    defaultStepOptions: {
+      cancelIcon: {
+        enabled: true,
+      },
+      scrollTo: {
+        behavior: "smooth",
+        block: "center",
+      },
+      popperOptions: {
+        modifiers: [{ name: "offset", options: { offset: [0, 12] } }],
+      },
+    },
+    useModalOverlay: true,
+  };
+
+  return (
+    <ShepherdTour steps={guideSteps} tourOptions={tourOptions}>
+      <Button />
+    </ShepherdTour>
+  );
+};
+
+const introduction = {
+  title: "Guide to Issue Certificate Hash",
+  text: `
+      <p> After wrapping the documents and obtaining a certificate hash (merkle root), the documents are ready to be issued on the document store smart contract. </p>
+      <p> In this guide, we will show you step by step instructions to issue the wrapped document.</p>
```

Comment:
to issue the certificates or the certificate hash

wrapped document is coming out of nowhere here

Reply:
updated.